.github/workflows/publish_charm.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - track/*
```

Comment:
Is it intended to publish a track charm to `latest/edge`? Shouldn't this be different for each track?

Reply:
Good catch. Updated